Pipfile

```diff
@@ -14,7 +14,7 @@ black = "==20.8b1"
 pytest = "~=6.2"
 pytest-cov = "~=2.11"
 
-tox = "~=3.21"
+tox = "~=3.22"
```

Comment:
unitTest passed, so it's no problem.

https://dev.azure.com/AppiumCI/Appium%20CI/_build/results?buildId=13966&view=results